Gifski/Gifski.swift

```diff
@@ -178,14 +178,29 @@ final class Gifski {
 
 			let generator = AVAssetImageGenerator(asset: asset)
 			generator.appliesPreferredTrackTransform = true
-			generator.requestedTimeToleranceBefore = .zero
-			generator.requestedTimeToleranceAfter = .zero
+
+			// `.zero` tolerance is much slower and fails a lot on macOS 11. (macOS 11.1)
+			if #available(macOS 11, *) {
+				generator.requestedTimeToleranceBefore = CMTime(seconds: 0.1, preferredTimescale: .video)
```

Comment:
@kornelski Do you think it's safe to do 0.1 threshold here? Maybe it should be based on the requested FPS, but not exactly sure what heuristics to use.

Reply:
that seems risky above 10fps. I'm worried that decoder could take advantage of it to snap to keyframes, so you'd have stutter around them.

I'd use `0.5/fps`, so that it's at worst half of delay in either direction.

Reply:
> I'd use 0.5/fps, so that it's at worst half of frame time in either direction.

👍